frontend/src/components/InGame/InGame.vue

```diff
@@ -1,9 +1,13 @@
 <template>
   <div>
-    <template v-if="userLocked">
-      <h2 class="mt-5">📡 Connecting …</h2>
-      <h6>Please be patient, this can take a couple of seconds.</h6>
-    </template>
+    <div v-if="error" class="mt-5">
```

Comment:
Do we still check if the user is locked or will it return an error?

Reply:
It will render an error message. However currently it’s unlikely to happen as the browser correctly end the connection on close.